RetailSuiteAccounting/UserInterface/01d_OpenItems.md

```diff
@@ -0,0 +1,226 @@
+
+- ![Left arrow](../../Assets/Icons/Back02.png "[Left arrow]") (Left arrow)   
+Click this arrow to open the *Post extras* side bar. For detailed information about searching in the *Accounting* module, see [Search in accounting](../Operation/14_SearchAccounting.md).
+
+- ![Previous](../../Assets/Icons/Previous.png "[Previous]")  *Scroll* ![Next](../../Assets/Icons/Next.png "[Next]")  
+Your can use the arrow buttons to scroll through the available accounts to search for a specific one.
+
+
+- *Seq. no.*  
+This column displays the sequential number.
+
+  > [Info] The posting numbers must be sequential and are added by the system automatically.
+
+- *Date*  
+This column displays the posting date. The valid date format is DDMMYY.
+
+- *BP*  
+This column displays the posting period (month).
+
+- *Jnl*  
+This column displays the journal.
+
+- *M*  
+This column displays a checkbox to select the open items to be cleared manually if necessary.
+
+  > [Info] Manually cleared postings are marked with an asterisk next to the checkbox.
+
+- *Contra account*  
+This column displays the contra account number.
+
+- *Tax key*  
+This column displays the tax key linked to the contra account, if any.
+
+- *Receipt1*  
+This column displays the receipt 1 value, usually the invoice number.
+
+  > [Info] The information provided in the *Receipt 1* field is essential, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
+
+- *Receipt2*  
+This column displays the receipt 2 value, which can be any other reference number, such as the order number.
+
+- *Cst1*  
+This column displays the cost unit number selected.
+
+  > [Info] This is only applicable if at least one cost center has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).
+
+- *Cst2*  
+This column displays the cost unit number selected.
+
+  > [Info] This is only applicable if at least one cost unit has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).
+
+- *Debit*  
+This column displays the debit amount of the selected account.
+
+- *Credit*  
+This column displays the credit amount of the selected account.
+
+- *(Unnamed column)*  
+This column displays a ![B](../../Assets/Icons/Beleg.png "[B]") icon if a receipt has been linked to the posting. When clicking the icon, the linked receipt is displayed in the *Attached document* window. For detailed information, see [Attached document](#attached-document).
+
+- *Text*  
+This column displays the posting text, if any has been entered.
+
+- *(Account number) (account name) [currency]*  
+This field introduces a list of all existing open items for this account.
+
+- *Balance (currency)*  
+This field displays the balance for each existing open item in the preceding account.    
```

Comment:
Kein Unterschied in der Liste zu den andern offenen Posten oder? Daher komplett löschen.